README.md

```diff
@@ -0,0 +1,34 @@
+# Docker images for opensnp.org
+
+Currently two images - a *opensnp.org-docker* image and a *dev* image. The *opensnp.org-docker* image in the root folder pulls all gems and other dependencies. The *dev* image adds a few dummy files so that you can run the image in a local container.
+
+# How to build the images
+
+In a screen session:
```

Comment:
I use screen a lot because I use several clusters that don't all have tmux installed, and I can't change anything on these clusters... So I use screen out of muscle-memory. That reminds me, screen isn't getting installed in the Dockerfile :neckbeard: